R/pokehelperr.R

```diff
@@ -20,7 +23,33 @@
 #' get_types(list('Pikachu', 'Eevee', 'Charizard', ...))
 #'
 get_types <- function(pokemon_names) {
-  # Function code (TBD in Milestone 3)
+   url <- "https://raw.githubusercontent.com/UBC-MDS/pokehelpyer/main/data/pokemon.csv"
```

Comment:
It is probably best to use a local copy of the data frame to improve efficiency. For example, see https://github.com/UBC-MDS/pokehelperr/pull/21/files#r1089860169.